packages/graph/hash_graph/lib/graph/src/store/postgres/knowledge/entity/mod.rs

```diff
@@ -46,31 +46,55 @@ impl<C: AsClient> PostgresStore<C> {
         entity_vertex_id: EntityVertexId,
         dependency_context: &'a mut DependencyContext,
         subgraph: &'a mut Subgraph,
-        current_resolve_depth: GraphResolveDepths,
+        mut resolve_depths: GraphResolveDepths,
```

Comment:
I somewhat liked the previous definition of `current_resolve_depth`(s) because it had some implication about it being temporary (as in "current"), and considering we mutate the heck out of these depths as we pass them in, this feels a bit like an unnecessary change.

I don't feel strongly, not asking for it to be changed back, just thought I'd highlight it.

Reply:
I hear you and the reasoning makes sense. Happy to revert it

Reply:
Reverted (and added an `s`) in ec9a2c45b28142e03fabb6bfa62a5ea8ecdad598